test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+odoo-addon-mrp_flattened_bom_xlsx_direct_materials_cost@git+https://github.com/OCA/manufacture-reporting.git@refs/pull/116/head#subdirectory=setup/mrp_flattened_bom_xlsx_direct_materials_cost
```

Comment:
@PauBForgeFlow Could remove the test-requirements and rebase your PR? All dependencies are now merged